content/_data/changelogs/lts.yml

```
@@ -9736,9 +9736,8 @@
 - version: "2.426.1"
   date: 2023-11-15
   banner: >
-      Short tags (without "jdk" in them) such as <code>jenkins/jenkins:2.414.3-alpine</code> are using Java 17 and not Java 11 like previously.
-      If you need to keep using Java 11, use tags like <code>jenkins/jenkins:2.414.3-jdk11</code>.
-      Also note that two new tags (2.414.3-alpine-jdk17 & 2.414.3-slim-jdk17) have been published without any content change a week later than the original ones.
+      Short container image tags (without "jdk" in them) such as <code>jenkins/jenkins:2.426.1</code> are now using Java 17.
+      If you need to continue using Java 11, use tags like <code>jenkins/jenkins:2.4264.2-jdk11</code>.
```

Sorry, bad suggestion on my part.
```suggestion
      If you need to continue using Java 11, use tags like <code>jenkins/jenkins:2.426.1-jdk11</code>.
```